scripts/enter-env.sh

```diff
@@ -64,7 +64,7 @@ if [ -f /etc/udev/rules.d/99-hdmi2usb-permissions.rules -o -f /lib/udev/rules.d/
 	true
 else
 	echo "Please install the HDMI2USB udev rules."
-	echo "These are installed by scripts/download-env-root.sh"
+	echo "These are installed by scripts/debian-setup.sh"
```

Comment:
Maybe reword this to "On debian, these can be installed by using scripts/debian-setup.sh"?

Reply:
Done. Also offering hint about the already-existing `HDMI2USB_UDEV_IGNORE` environment variable.

I'm surprised the Travis reported failure. This can't be caused this PR changes, can it?

Reply:
https://travis-ci.org/timvideos/litex-buildenv/builds/597670776 seems to mean that only some targets fail.

Started to fail on commit 534513f2fd6a2f5924c3d3f46237c5db946a856e which also has unrelated changes.

Is something broken on the CI scripts?